src/components/CheckboxGroup/CheckboxGroup.test.tsx

```diff
@@ -58,6 +58,16 @@ describe("CheckboxGroup Accessibility", () => {
     );
     expect(await axe(container)).toHaveNoViolations();
   });
+  it("passes axe accessibility with the legend hidden", async () => {
```

Comment:
This is essentially the same test as the first one. Maybe we need more comments in the first one instead of repeating the test to explain what we are looking for in terms of a11y testing.

Reply:
Well, of course, I am an idiot and I did not set `showLabel` to `false` for this test.  How about now?